src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const MEDIUM_TERM = 3;
+  const MEDIUM_TERM_DISCOUNT = 20;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const PRICE = 40;
+
+  let totalPrice = days * PRICE;
```

Comment:
![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/96409563/a307b5d1-cf29-479e-8973-a1922c444452)
You don't reassign this variable, no need to make it let